boilerplate.py

```diff
@@ -209,7 +210,15 @@ def format_value(value):
                 mappable = ''
 
             # Get argspec of wrapped function
-            args, varargs, varkw, defaults = inspect.getargspec(getattr(Axes, func))
+            base_func = getattr(Axes, func)
+            has_data = 'data' in inspect.signature(base_func).parameters
+            if hasattr(base_func, '__wrapped__'):
```

Comment:
this should probably use inspect.unwrap(...) ([source](https://github.com/python/cpython/blob/d15ca79a9284ab5e1d8fb552e31955a1092f339c/Lib/inspect.py#L447))